card/lib/card/format/render.rb

```diff
@@ -71,7 +71,7 @@ def before_view view
       end
 
       def voo
-        @voo
+        @voo ||= View.new self, nil, {}
```

Comment:
I feel like there was a reason we didn't do this before, but I can't remember it.  We should certainly remember this in case there is any caching weirdness.